test_main_import.py

```
@@ -0,0 +1,13 @@
+import sys
+import os
+
+# Add repo root to sys.path
+sys.path.insert(0, os.getcwd())
+
+try:
+    import backend.main
+    print("Successfully imported backend.main")
+except ImportError as e:
+    print(f"Failed to import backend.main: {e}")
+except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)